### ISSUE TITLE
[Visual requirement – Make Code – Home Page] Tooltip is not defined for visually appearing (See More & See Previous) icon controls on ‘Home Page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3437]
**User Experience:**
User would find difficulty to identify the purpose of icon control if tooltip is not defined for visual icon and also user won't be able to perform functionality over that icon control.
 
**Repro-Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify tooltip of visual icon (See More & See Previous) controls.
 
**Actual Result:**
Tooltip is not defined for visually appearing (See More & See Previous) icon controls on ‘Home Page’, due to this user is unable to identify the purpose of icon control and also won't be able to take action on such controls.

**Expected Result:**
Tooltip should be defined as (See More & See Previous) for visually appearing icon control (</>) on ‘Home Page’, so that user could easily identify the purpose of icon control and perform functionality over that control.
 
**Note:** 
Same issue exists on ‘Chrome’ &  Firefox.

**MAS Reference:**
[MAS 1.3.3 - Sensory Characteristics](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358})

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
#URL:https://makecode.microbit.org/#
